apprentice_teacher.rb

```diff
@@ -1,22 +1,23 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'Instructors'
```

Comment:
case needs to match the filename (not sure if this is true across all systems, but it gave errors on mine).